src/SiteMaster/Core/Auditor/Logger/Links.php

```diff
@@ -0,0 +1,50 @@
+<?php
+namespace SiteMaster\Core\Auditor\Logger;
+
+use DOMXPath;
+use SiteMaster\Core\Auditor\Site\Page;
+use SiteMaster\Core\Util;
+
+class Links extends \Spider_LoggerAbstract
+{
+    /**
+     * @var bool|\Spider
+     */
+    protected $spider = false;
```

Comment:
Fixed in https://github.com/mfairchild365/site_master/commit/ce09c1e7155e0082bb696e8e2feab3d1ae38c06a